File: Python/practical/14.py
# Program to find the location of a coordinate point

(x, y) = map(int, input("Enter the coordinates: ").split())

if x == 0 and y == 0:
    print(f"The coordinate ({x}, {y}) is the origin")
elif x >= 0 and y == 0:
    print(f"The coordinate ({x}, {y}) is on the x axis right to the origin")
elif x <= 0 and y ==0:
    print(f"The coordinate ({x}, {y}) is on the x axis left to the origin")
elif x == 0  and y >= 0:
    print(f"The coordinate ({x}, {y}) is on the y axis above the origin")
elif x == 0 and y <= 0:
    print(f"The coordinate ({x}, {y}) is on the y axis below the origin")
elif x >= 0 and y >= 0:
    print(f"The coordinate ({x}, {y}) is in the first quadrant")
elif x <= 0 and y >= 0:
    print(f"The coordinate ({x}, {y}) is in the second quadrant")
elif x <= 0 and y <= 0:
    print(f"The coordinate ({x}, {y}) is in the third quadrant")
elif x >= 0 and y <= 0:
    print(f"The coordinate ({x}, {y}) is in the fourth quadrant")

File: C/SEM I/Practical/Done/14.c
/* Write a program to accept a coordinate point in a XY coordinate system and determine in which quadrant the coordinate point lies */

#include <stdio.h>
#include <conio.h>
#include <stdlib.h>

int count = 0;

int main(void) {
	int rep = 1;

	while (rep == 1) {
		float coordX, coordY;

		printf("Enter the abscissa of the coordinate: ");
		scanf("%f", & coordX);
		printf("Enter the ordinate of the coordinate: ");
		scanf("%f", & coordY);

		if (coordX > 0 && coordY > 0) {
			printf("The point (%.2f, %.2f) is in the first quadrant\n", coordX, coordY);
		} else if (coordX < 0 && coordY > 0) {
			printf("The point (%.2f, %.2f) is in the second quadrant\n", coordX, coordY);
		} else if (coordX < 0 && coordY < 0) {
			printf("The point (%.2f, %.2f) is in the third quadrant\n", coordX, coordY);
		} else if (coordX > 0 && coordY < 0) {
			printf("The point (%.2f, %.2f) is in the fourth quadrant\n", coordX, coordY);
		} else if (coordX == 0 && coordY > 0) {
			printf("The point (%.2f, %.2f) is on the Y axis above the origin\n", coordX, coordY);
		} else if (coordX == 0 && coordY < 0) {
			printf("The point (%.2f, %.2f) is on the Y axis below the origin\n", coordX, coordY);
		} else if (coordX > 0 && coordY == 0) {
			printf("The point (%.2f, %.2f) is on the X axis right to the origin\n", coordX, coordY);
		} else if (coordX < 0 && coordY == 0) {
			printf("The point (%.2f, %.2f) is on the X axis left to the origin\n", coordX, coordY);
		}

		printf("\nRepeat Program?: ");
		scanf("%d", & rep);

		count++;

		if (count >= 50) {
			printf("Abnormal END\n");

			getch();
			exit(0);
		}

	}

	printf("Normal END\n");

	getch();
	return 0;
}